modules/base/src/datavisualizer/imageinformationvisualizer.cpp

```diff
@@ -76,26 +76,28 @@ bool ImageInformationVisualizer::hasSourceProcessor() const { return true; }
 bool ImageInformationVisualizer::hasVisualizerNetwork() const { return true; }
 
 std::pair<Processor*, Outport*> ImageInformationVisualizer::addSourceProcessor(
-    const std::filesystem::path& filename, ProcessorNetwork* net) const {
+    const std::filesystem::path& filename, ProcessorNetwork* net, const ivec2& initialPos) const {
 
-    auto source = net->addProcessor(util::makeProcessor<ImageSource>(GP{0, 0}, app_, filename));
```

Comment:
This line will no longer compile since you now require explicit cast of GP...

Reply:
I am aware of this. Complain to the linter.

Reply:
one can overrule the linter...